hadoop-master1/Dockerfile

```diff
@@ -76,6 +76,7 @@ ADD download/trino-cli-${TRINO_VERSION}-executable.jar ${TRINO_HOME}/bin/trino-c
 
 RUN /opt/hadoop-init.d/init-hdfs.sh
 RUN /opt/trino-init.d/init-workdir.sh
+RUN /opt/ssh/init-ssh_config.sh
```

Comment:
=> ERROR [19/26] RUN /opt/ssh/init-ssh_config.sh                                                                                                                 0.2s
------
 > [19/26] RUN /opt/ssh/init-ssh_config.sh:
0.164 /bin/sh: line 1: /opt/ssh/init-ssh_config.sh: Permission denied
------
Dockerfile:79
--------------------
  77 |     RUN /opt/hadoop-init.d/init-hdfs.sh
  78 |     RUN /opt/trino-init.d/init-workdir.sh
  79 | >>> RUN /opt/ssh/init-ssh_config.sh
  80 |
  81 |     ADD download/log4j2-appender-nodep-${LOKI_APPENDER_VERSION}.jar ${HIVE_HOME}/lib/
--------------------
ERROR: failed to solve: process "/bin/sh -c /opt/ssh/init-ssh_config.sh" did not complete successfully: exit code: 126

@pan3793 It's strange, why other scripts do not need authorization.